pandas/tests/plotting/test_datetimelike.py

```diff
@@ -815,6 +815,23 @@ def test_mixed_freq_shared_ax(self):
         # self.assertEqual(ax1.lines[0].get_xydata()[0, 0],
         #                  ax2.lines[0].get_xydata()[0, 0])
 
+    def test_nat_handling(self):
+
+        import matplotlib.pyplot as plt  # noqa
+
+        fig = plt.gcf()
```

Comment:
Is this necessary for the test to work? Our base class should take care of closing all the figures, so there shouldn't be any hanging around from a previous test.